build_tools/mako/configuration.py

```diff
@@ -38,7 +46,13 @@ def __init__(self,
     self.mako_tag = mako_tag
     # This setup is going to be deprecated. Now we only test mhlo inputs for
     # Mako setup, hardcoded would make it eaiser.
-    self.compilation_flags = compilation_flags + ['--iree-input-type=mhlo']
+    self.compilation_flags = \
+      pipeline_options_to_string("iree-flow-transformation-pipeline",
+                                 flow_transformation_flags) + \
+                                 ["--iree-hal-transformation-pipeline"] + \
+                                 ["--iree-vm-transformation-pipeline"] + \
+                                 compilation_flags + \
+                                 ['--iree-input-type=mhlo']
```

Comment:
Do we miss `)` here?

You can run the commands to format the python file and it will raise errors if the syntax is not correct.

```
$ yapf configuration.py > a.py
$ mv a.py configuration.py
```

Reply:
I am looking into these errors based on the pre-merge checks.